### PR TITLE
Feature/adddate

### PR DESCRIPTION
**Pull request by akhilvuputuri**

### CODE REVIEW

File: src/main/java/seedu/eylah/diettracker/model/food/Calories.java [hunk 1]
```diff
@@ -25,6 +25,10 @@ public Calories(long calories) {
         this.value = calories;
     }
 
+    public long getValue() {
+        return value;
+    }
+
     /**
      * Returns true if a given string is a valid phone number.
      */
```
Comment (Jarrod-Bob) [inline src/main/java/seedu/eylah/diettracker/model/food/Calories.java]: Please edit this javadoc comment to reflect the actual usage.

File: src/main/java/seedu/eylah/diettracker/model/food/Date.java [hunk 2]
```diff
@@ -13,6 +17,23 @@ public Date() {
...
+        if (date == null || date == "") {
+            this.value = LocalDateTime.now();
+        } else {
+            this.value = LocalDateTime.parse(date);
+        }
+    }
+
+    public String getStorageString() {
+        return value.toString();
+    }
```
Comment (Jarrod-Bob) [inline src/main/java/seedu/eylah/diettracker/model/food/Date.java:35]: Clear and concise description of what the Date is.

**Review by Jarrod-Bob — CHANGES_REQUESTED** [review]
Do check on the failed build.

File: src/main/java/seedu/eylah/diettracker/model/food/Calories.java (hunk 1 above)
Comment (akhilvuputuri) [reply src/main/java/seedu/eylah/diettracker/model/food/Calories.java]: Alright thanks, changed!

**Review by Jarrod-Bob — APPROVED** [review]

**Pull request merged**